openapi/reputation/reputation.yaml

```diff
@@ -89,7 +89,7 @@ paths:
             type: string
           in: query
           name: 'page[cursor]'
-          description: 'The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links. [Pagination Docs](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg0-overview#paging).'
+          description: 'The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links. Please note that users can retrieve a maximum of 10,000 records due to paging limitations. [Pagination Docs](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg0-overview#paging).'
```

Comment:
```suggestion
          description: 'The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links. Please note that users can retrieve a maximum of 10,000 records in a single page due to paging limitations. [Pagination Docs](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg0-overview#paging).'
```

Reply:
Actually that is not correct. The total across all pages can not be more than 10000.

Reply:
Yeah, You're correct. I have updated it.